.gitignore

```diff
@@ -19,6 +22,7 @@ dist/
 __pycache__/
 htmlcov/
 *.egg-info/
+doc/**.ipynb
```

Comment:
Sorry I get back on this.
For me, this `.gitignore` in the root directory doesn't work as required. So, better to add
```gitignore
*.ipynb
```
to `doc/.gitignore`.